src/components/JobWizard/JobWizard.util.js

```diff
@@ -811,24 +811,26 @@ const generateDataInputs = dataInputsTableData => {
 }
 
 const generateEnvironmentVariables = envVarData => {
```

Comment:
```suggestion
const generateEnvironmentVariables = (envVarData = []) => {
```

Reply:
I believe there is no need to have ternary operator. You can have just a default value